pkg/ui/src/views/shared/components/loading/index.tsx

```diff
@@ -18,5 +18,9 @@ export default function Loading(props: LoadingProps) {
   if (props.loading) {
     return <div className={props.className} style={image} />;
   }
-  return props.children as JSX.Element;
+  return (
```

Comment:
It might be worth commenting on why this wrapper div is necessary so we don't accidentally remove it.  That comment might also mention that after upgrading to React 16 we could convert it to a fragment.